src/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>CATcher</title>
-  <base href="/">
+  <base href="./">
```

Comment:
Ideally, this line should remain as `<base href="/">` during development. Only when making a release then there is a need to change it. If the href is `./` during development, there might be some routing error when electron reloads during development. Have you encountered it?

Also just a suggestion, you can uncomment this line out to aid you in development. https://github.com/testathor/CATcher/blob/master/main.ts#L35

Reply:
Sure, i'll leave this commented out during development. I have not encountered any routing issues with the `./` enabled.